test/utils/vaultMath.test.ts

```diff
@@ -232,30 +232,30 @@ describe('maxCollateralForNewVault', () => {
   it('should prevent the user from exceeding the mint limit', () => {
```

Comment:
Confirmed this test now fails without change. We also have https://github.com/Agoric/dapp-inter/blob/6aa6db5ede8a9586d46fcd295dc50c78d1372ea6/test/utils/vaultMath.test.ts#L19 to test the round-up case.

Reply:
thanks for checking and noting